pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -0,0 +1,85 @@
+---
+title: Learn
+layout: wide_index
+social:
+  "og:description": "Anyone can learn the basic concepts of Computer Science. Choose an intro activity to start learning."
+  "og:image": "http://code.org/images/fit-520/codehoc3.jpg"
+  "og:image:width": 520
+  "og:image:height": 391
+  "og:video": ""
+  "og:video:width": ""
+  "og:video:height": ""
+  "og:video:type": ""
+---
+
+-# hourofcode.com uses an early version of Bootstrap 3 which didn't have this wider dialog.
+:css
+  @media (min-width: 992px)
+  {
+    .modal-lg {
+        width: 900px;
+  }
+
+#fullwidth{style: "background-color: #2799a4; overflow: hidden"}
+  = view :header
+
+  .banner
+    .container{style: "color: white"}
+      .col-xs-12.col-sm-6{style: "text-align: center"}
+        %img{src: CDO.code_org_url("/images/learn/hour-of-code-logo-2016.png"), style: "margin-top: 30px; margin-bottom: 30px; max-width: 100%; padding-left: 5px; padding-right: 5px;"}
+        %br/
+        The Hour of Code is a global movement reaching tens of millions of students in 180+ countries. Anyone, anywhere can organize an Hour of Code event. One-hour tutorials are available in over 45 languages for all ages.
+        %br/
+        %br/
+        %a{href: "/#join", target: "_blank"}
+          %button
+            Host an Hour
+        &nbsp;
+        &nbsp;
+        %a{href: "/how-to", target: "_blank"}
+          %button
+            Teacher Guide
+        %br/
+        %br/
+      #studentimage.col-xs-12.col-sm-6.hidden-xs{style: "height: 278px"}
+
+-# Not ideal but we can pull the undigested file from /blockly.
+%script{src: CDO.studio_url('/blockly/js/tutorialExplorer.js')}
+
+-# Load tutorial images from code.org so we don't have to duplicate them on hourofcode.com.
+- tutorials_contents = Tutorials.new(:tutorials).contents(request.host_with_port)
+- tutorials_contents.each do |tutorial|
+  - tutorial[:image] = CDO.code_org_url(tutorial[:image])
+
+- tutorials_contents.each do |tutorial|
+  - tutorial[:contact] = nil
+
+#tutorials
+
+.clear{style: "clear: both"}
+
+:javascript
+
+  $(document).ready(function() {
+    // Send through some values that the JavaScript will need.
+    var tutorialExplorerManager = new window.TutorialExplorerManager({
+      tutorials: #{tutorials_contents.to_json},
+      locale: "#{request.locale}",
+      roboticsButton: true,
+      legacyLink: "#{CDO.code_org_url('/learn/legacy')}"
+      }
+    );
+    tutorialExplorerManager.renderToElement(document.getElementById('tutorials'));
+
+    // Pick a random student image (1-2 inclusive) and fade it in.
+    var studentImageIndex = Math.floor(Math.random() * 2) + 1;
+    var studentImage = "#{CDO.code_org_url('/images/learn/student')}" + studentImageIndex + '.png';
+
+    var img = $('<img>');
+    img.attr('src', studentImage);
+    img.attr('style', 'opacity: 0');
+    img.appendTo('#studentimage');
+    img.on('load', function() {
+      img.fadeTo('normal', 1);
+    });
+  });
```

Comment:
This is starting to look like more code than I'd like to see in a haml file.

We came up with a cleaner approach for getting data from dashboard/haml to JS in https://github.com/code-dot-org/code-dot-org/pull/11322. Probably worth doing something like that here, but it might make sense to do that as a separate PR to make things less confusing. Happy to help you out to get that working.